server/src/test/java/org/elasticsearch/index/IndexVersionTests.java

```diff
@@ -151,8 +151,6 @@ public void testMax() {
         }
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA)
-    @AwaitsFix(bugUrl = "believe this fails because index version has not yet been bumped to 9.0")
     public void testMinimumCompatibleVersion() {
         assertThat(IndexVersion.getMinimumCompatibleIndexVersion(7170099), equalTo(IndexVersion.fromId(6000099)));
         assertThat(IndexVersion.getMinimumCompatibleIndexVersion(8000099), equalTo(IndexVersion.fromId(7000099)));
```

Comment:
We should have a new assert for the v9 index version

Reply:
Added